src/doctopus/db/schema.clj

```diff
@@ -0,0 +1,57 @@
+(ns doctopus.db.schema
+  (:require [clojure.java.jdbc :as sql]
+            [taoensso.timbre :as log]
+            [doctopus.configuration :refer [server-config]]))
+
+(defn- get-subname
+  "pull database options out of the local config"
+  []
+  (let [{host :host port :port db :db} (:database (server-config))]
+    (str "//" (or host "localhost") ":" (or port "5432") "/"
+         (or db "doctopus"))))
```

Comment:
May I suggest: if we change this to be a function we call with `server-config` as an arg, we can use map-destructuring syntax:

```
(defn- get-subname
  "pull database options out of the local config"
  [{:keys [host port db] :or {host "localhost" port 5432 db "doctopus"}}]
  (let [tpl "//%s:%d/%s"]
    (format tpl host port db)))

=> (get-subname {})
"//localhost:5432/doctopus"
=> (get-subname {:db "poot"})
"//localhost:5432/poot"
```